package.json

```diff
@@ -75,7 +75,7 @@
   },
   "scripts": {
     "test": "grunt test",
-    "e2e": "./node_modules/.bin/codeceptjs run --steps --reporter mochawesome",
```

Comment:
OK. Regarding the configs, it would be good to have a way to select the env of the application under test. By default, would be localhost but if I wanted to execute the tests on dev or test server I would load different settings. This would come in handy in CI settings.

Reply:
I agree. That's out of the scope of these changes, but a great idea. How about a flag passed into e2e. Maybe ```npm run e2e --server dev``` or something like that?